x/concentrated-liquidity/store.go

```diff
@@ -48,18 +57,6 @@ func ParsePositionIdFromBz(bz []byte) (uint64, error) {
 	return sdk.BigEndianToUint64(bz), nil
 }
 
-// ParsePosition unmarshals a byte slice into a model.Position struct.
-// Returns an error if the byte slice is empty.
-// Returns an error if fails to unmarshal.
-func ParsePosition(value []byte) (model.Position, error) {
-	position := model.Position{}
-	err := proto.Unmarshal(value, &position)
-	if err != nil {
-		return model.Position{}, err
-	}
-	return position, nil
-}
-
```

Comment:
nit: why not have it as a function and not callback in case we need to reuse it later?

Reply:
Added it back here https://github.com/osmosis-labs/osmosis/pull/4724/commits/31eba27a87469b33a9c878cc31d92c8b51377055